source/newcommitter.md

```diff
@@ -20,11 +20,46 @@ The [Contributor Ladder](https://community.apache.org/contributor-ladder.html) h
 
 {{% toc %}}
 
+<a name="NewCommitter-Summary"></a>
+### TL;DR - Inviting a new Committer
+
```

Comment:
First, find the name of the person to be discussed. We do not invite github ids but people into our communities.

Reply:
That sentence, phrased almost that way, appears in the detail section of the document. I guess I'm unclear exactly what change you're suggesting.